JASP-R-Interface/jaspResults/R/writeImage.R

```diff
@@ -29,23 +29,29 @@ writeImageJaspResults <- function(width=320, height=320, plot, obj = TRUE, relat
   setwd(root)
   on.exit(setwd(oldwd))
 
+  # Operating System information
+  type <- "cairo"
+  if (Sys.info()["sysname"]=="Darwin"){
+    type <- "quartz"
+  }
   if (ggplot2::is.ggplot(plot) || inherits(plot, c("gtable", "ggMatrixplot", "JASPgraphs"))) {
     ppi <- .fromRCPP(".ppi")
 
-    # fix for mac
-    if (Sys.info()["sysname"] == "Darwin") ppi <- ppi / 2
-
-    ggplot2::ggsave(relativePathpng, plot, "png",
-                    width  = 1.5*width/ppi,
-                    height = 1.5*height/ppi,
-                    dpi    = 2*ppi,
-                    bg     = "transparent")
+    pngMultip <- .fromRCPP(".ppi") / 96
+    ggplot2::ggsave(
```

Comment:
I think this is a good idea, although it does (re)introduce the following 2 problems:
1. The density plot now has different line thickness than the bar plot in descriptives (should be easy to fix for this plot)
2. y axis label letters are slightly cut off on windows with 100% (default) scaling (look at the `n` and the `u` in `counts`; it's less prominent in this anti-aliased png screenshot than in JASP)

![afbeelding](https://user-images.githubusercontent.com/11596858/47072924-6667b500-d1f7-11e8-8eeb-5d1486cd67d0.png)